gef.py

```diff
@@ -1717,20 +1724,28 @@ def is_branch_taken(self, insn):
                 if (op & 1<<i) == 0: taken, reason = True, "{}&1<<{}==0".format(reg,i)
                 else: taken, reason = False, "{}&1<<{}!=0".format(reg,i)
 
-        elif mnemo.endswith("eq"): taken, reason = val&(1<<flags["zero"]), "Z"
-        elif mnemo.endswith("ne"): taken, reason = val&(1<<flags["zero"]) == 0, "!Z"
+        elif mnemo.endswith("eq"): taken, reason = bool(val&(1<<flags["zero"])), "Z"
```

Comment:
These all should be the same as implemented in `ARM` right? Is there a way we could avoid duplicating the logic?

Reply:
Yes, we should look into that. I am sure I already have looked into it and gave up, and I probably won't do it for this PR/release, but it's worth looking into.